src/qc/calculate_qc_metrics/config.vsh.yaml

```diff
@@ -0,0 +1,88 @@
+functionality:
+  name: calculate_qc_metrics
+  namespace: "qc"
+  description: |
+    Add basic quality control metrics to an .h5mu file.
+
+    The metrics are comparable to what scanpy.pp.calculate_qc_metrics output,
+    although they have slightly different names:
+
+    Var metrics (name in this component -> name in scanpy):
+      - pct_dropout -> pct_dropout_by_{expr_type}
+      - num_nonzero_obs -> n_cells_by_{expr_type}
+      - obs_mean -> mean_{expr_type}
+      - total_counts -> total_{expr_type}
+
+    Obs metrics:
+      - num_nonzero_vars -> n_genes_by_{expr_type}
+      - pct_{var_qc_metrics]} -> pct_{expr_type}_{qc_var}
+      - total_counts_{var_qc_metrics} -> total_{expr_type}_{qc_var}
+      - pct_of_counts_in_top_{top_n_vars}_vars -> pct_{expr_type}_in_top_{n}_{var_type}
+      - total_counts -> total_{expr_type}
+      
+  authors:
+    - __merge__: ../../authors/dries_schaumont.yaml
+      roles: [ author ]
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: "--input"
+          type: file
+          description: Input h5mu file
+          direction: input
+          required: true
+          example: input.h5mu
+        - name: "--modality"
+          type: string
+          default: "rna"
+          required: false
+        - name: "--layer"
+          type: string
+          example: "raw_counts"
+          required: false
+        - name: "--var_qc_metrics"
+          description: |
+            Keys to select a boolean (containing only True or False) column from .var.
+            For each cell, calculate the proportion of total values for genes which are labeled 'True', 
+            compared to the total sum of the values for all genes.
+          type: string
+          multiple: True
+          multiple_sep: ','
+          example: "ercc,hvg_genes"
```

Comment:
Imo `hvg_genes` should be simply `hvg`, `highly_variable` or `highly_variable_genes`.